drivers/sensor/ens210/Kconfig

```diff
@@ -10,6 +10,32 @@ menuconfig ENS210
 	  Enable driver for ENS210 Digital Temperature and Humidity sensor.
 if ENS210
 
+choice
+	prompt "Temperature measurement mode"
+	default ENS210_TEMPERATURE_CONTINUOUS
+	help
+	  Enable/disable temperature measurements and set measurement mode.
+config ENS210_TEMPERATURE_OFF
```

Comment:
I think there is no need for off. You can select the channel you want to fetch.

Reply:
Well, if I want to use the sensor to only measure relative humidity or only measure temperature, then it makes sense. That's why I added the possibility to switch them off as well.

Reply:
Right. I forgot about the "run".